README.md

```diff
@@ -72,17 +72,14 @@ will build and install *stravalib* in your *pip* package repository.
 
 To execute **unit tests** you will need to run
 ```
-shell$ nosetests
+shell$ pytest stravalib/tests/unit
 ```
-or
-```
-shell$ nosetests-3
-```
-if you use Python3.
 
 To run **integration** tests you will need to rename *test.ini-example* (which you can find *<your-root-proj-dir>*/stravalib/tests/) to *test.ini*
```

Comment:
is this replaced by the mock element? i haven't dug into the actual tests yet :)

Reply:
I still need to modify the documentation in a.o. README.md, based on this review.